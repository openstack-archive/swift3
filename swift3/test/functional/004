#!/bin/bash

# HEAD Bucket

. ./common

S3USER=tester

_s3_put /bucket

_s3_head /bucket | _filter_curl

