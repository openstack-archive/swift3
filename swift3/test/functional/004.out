QA output created by 004
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -I -X HEAD http://SWIFT_HOST/bucket... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/plain; charset=utf-8
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
x-rgw-bytes-used: 0
x-rgw-object-count: 0
X-Trans-Id: TXID

